bikeshed/MetadataManager.py

```diff
@@ -744,6 +745,7 @@ def joinDdList(a,b):
     "Repository": Metadata("Repository", "repository", joinValue, parseRepository),
     "Revision": Metadata("Revision", "level", joinValue, parseLevel),
     "Shortname": Metadata("Shortname", "shortname", joinValue, parseLiteral),
+    "Show Hide Deleted Button": Metadata("Show 'Hide deleted text' button", "showHideDeletedButton", joinValue, parseBoolean),
```

Comment:
That's not accurate. I'm thinking: `Toggle Diffs`?  Oh, and the key and first argument have to be identical; it's unfortunately repeated information, but I can't remove the duplication cleanly so eh.